src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
 // write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => reject(new Error()), 3000);
+});
+
+function successHandler() {
+  const div = document.createElement('div');
+
+  div.setAttribute('class', 'message');
+  div.textContent = 'Promise was resolved!';
+  document.body.append(div);
+}
+
+function errorHandler() {
+  const div = document.createElement('div');
+
+  div.setAttribute('class', 'message error-message');
+  div.textContent = 'Promise was rejected!';
+  document.body.append(div);
+}
+
+promise1.then(successHandler).catch(errorHandler);
+promise2.then(successHandler).catch(errorHandler);
```

Comment:
Currently, `promise1` and `promise2` are handled separately, which means the error handler for `promise2` will always execute after 3 seconds, even if `promise1` resolves. Consider using `Promise.race([promise1, promise2])` to handle the first settled promise, or `Promise.allSettled([promise1, promise2])` if you want to handle both outcomes together.